usr/share/rear/layout/save/default/445_guess_bootloader.sh

```diff
@@ -69,7 +69,9 @@ for block_device in /sys/block/* ; do
         # but actually another non-EFI bootloader is used here
         # cf. https://github.com/rear/rear/issues/1752#issue-303856221
         # so that in the 'Hah!IdontNeedEFI' case only non-EFI bootloaders are tested:
-        for known_bootloader in GRUB2 GRUB ELILO LILO ; do
+	# cf. https://github.com/rear/rear/issues/2137
+	#  s390 - zipl boot loader for rhel, ubuntu
+        for known_bootloader in GRUB2 GRUB ELILO LILO zIPL ; do
```

Comment:
Only a side note regarding the spelling of the bootloader values:

Up to now we use only uppercase for the bootloader values,
see the various `... | tr '[a-z]' '[A-Z]' >$bootloader_file` commands in
usr/share/rear/layout/save/default/445_guess_bootloader.sh

I don't know a reason why we use only uppercase for the bootloader values
but for consistency I would suggest to also use uppercased `ZIPL`.

Reply:
completed.

Reply:
got the networking working.  
problem 
  ethtool -P $network_interface -> on s390 returns 00:00:...
  ifconfig returns a mac
     cat /sys/devices/qeth/0.0.0610/net/enccw0.0.0610/address -> returns mac

put a simple fix in while i look to see why the problem occurs